src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -114,7 +114,7 @@ public static void MatTimesSrc(bool tran, AlignedArray mat, AlignedArray src, Al
             else if (Sse.IsSupported)
             {
                 Contracts.Assert(crun <= dst.Size);
-                SseIntrinsics.MatMulPA(mat, rgposSrc, srcValues, posMin, iposMin, iposLim, dst, crun, srcValues.Size);
+                SseIntrinsics.MatMulP(mat, rgposSrc, srcValues, posMin, iposMin, iposLim, dst, crun, srcValues.Size);
```

Comment:
The corresponding `AvxIntrinsics` and `SseIntrinsics` methods should be named the same. This one is `MatMulP`, while the Avx one is `MatMulPX`.